frontend/document/djvudocument.lua

```diff
@@ -54,11 +54,20 @@ function DjvuDocument:updateColorRendering()
 end
 
 function DjvuDocument:getProps()
+    local props = self._document:getMetadata()
```

Comment:
@xelxebar Note the unit tests:

```
frontend/document/djvudocument.lua:57: attempt to call method 'getMetadata' (a nil value)

stack traceback:
	frontend/document/djvudocument.lua:57: in function 'getProps'
	frontend/apps/reader/readerui.lua:375: in function 'init'
	frontend/ui/widget/widget.lua:48: in function 'new'
	spec/front/unit/readerpaging_spec.lua:76: in function <spec/front/unit/readerpaging_spec.lua:74>
```

Reply:
Yeah, this only works with  koreader/koreader-base#753.
I'm not exactly sure how to deal with pull requests like this that are really split between base. Would it be better to just wait until base integrates?

Reply:
> Would it be better to just wait until base integrates?

Yes, and that's needed: we'll merge your koreader-base PR first.

Once that is done, you'll need to add base to this PR. I usually do it this way:
Go into your koreader local repo.
```
git submodule init base (only needed the first time if not yet done)
git submodule update --remote base
git status (you'll see a diff showing the old and new hash for koreader-base)
git add base
git commit (message "bump base")
git push origin yourbranch
```
And we'll squash the multiple commits into a single one (if you have not done a rebase to add base to your first commit).

Reply:
Okay. Once base is hammered out, I'll fix this guy up. Yeah, I'll just rebase and `commit --amend`.